composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "sparkpost/php-sparkpost",
     "description": "SDK for interfacing with SparkPost APIs",
     "license": "Apache 2.0",
+    "version": "0.2.0",
```

Comment:
I'm not sure, I want to talk to @richleland and @aydrian about that.  I personally think it should be just because this breaks everything existing. But since its still a 0.* I'm not sure, we may be ok with that.